### ISSUE TITLE
import/namespace doesn't work with export namespace statements 

### ISSUE THREAD

**Issue by NickHeiner** [import-js/eslint-plugin-import#2161]
There's a chance that I'm misunderstanding the syntax and everything is fine.

```js
// upstream.js
export const a = 1;

// middle.js
export * as myName from './upstream'

// downstream.js
import * as middle from './middle';

// 3:20 error  'myName' not found in imported namespace 'middle'  import/namespace
console.log(middle.myName);
```

I believe that `middle.myName` is valid, and should not be flagged as an error.

Using version 2.23.4.

**Comment by ljharb** [comment]
I agree, this should be valid.

**Comment by ljharb** [comment]
Hmm, my test case passes for this one locally. Can you try v2.24.0?

**Comment by NickHeiner** [comment]
Hmm, it still is happening for me using v2.24.0. Repro repo: https://github.com/NickHeiner/repro-eslint-plugin-import-2161. Run `yarn test` to see the issue.

**Comment by ljharb** [comment]
That calls into `https://artifacts.netflix.net/api/npm/npm-netflix/eslint/-` which i can't access, but i think i can update the yarn.lock to point to npm.

**Comment by NickHeiner** [comment]
🤦 AMATEUR HOUR OVER HERE.

I updated the `yarn.lock` file.

**Comment by ljharb** [comment]
definitely can repro it with your repo; the exact same copy of eslint-plugin-import can't repro it yet in the tests :-(

**Comment by NickHeiner** [comment]
Ah, it's so tricky when that happens!

**Comment by ljharb** [comment]
Released in v2.24.1
